### PR TITLE
add requestType to correctly label fetch requests in telemetry

### PR DESCRIPTION
**Pull request by markwolff**
All auto tracked requests are listed as `Ajax` in portal. Now that we support collection of fetch requests, we should label it properly.

- rename `commandName` legacy code
- Add requestType to correctly label fetch requests

TODO:
- [ ] Add tests

### CODE REVIEW

File: extensions/applicationinsights-dependencies-js/src/ajax.ts [hunk 1]
```diff
@@ -541,6 +542,7 @@ export class AjaxMonitor implements ITelemetryPlugin, IDependenciesPlugin, IInst
                     id: ajaxData.id,
                     target: ajaxData.getAbsoluteUrl(),
                     type: ajaxData.getPathName(),
+                    requestType: "Fetch",
```
Comment (hectorhdzg) [inline extensions/applicationinsights-dependencies-js/src/ajax.ts]: So Type and requestType are different things?, this doesn't look good in Ajax you sue type and in Fetch calls you use both requestType and type, is this new field documented anywhere?

Comment (markwolff) [reply extensions/applicationinsights-dependencies-js/src/ajax.ts]: type is existing field. Would prefer to name this type. The default value of `requestType` is `"Ajax"`, but it should probably be explicitly set here.

Comment (markwolff) [reply extensions/applicationinsights-dependencies-js/src/ajax.ts]: Also I think `type` is old `commandName`

**Comment by markwolff** [comment]
Closing for now. Will redo in another PR
